README.md

```diff
@@ -1,20 +1,26 @@
 # CAN-Explorer
 An application to visualize CAN data and interact with the CAN bus
 
+__As of Jan 23__ 
+
+Make sure you have pulled the most recent version of [uwmidsun/box](https://github.com/uw-midsun/box) if you haven't done so already and run `vagrant reload && vagrant ssh` to apply new port-forwarding settings
+
 SSH into the vagrant box and run
 ```
+cd shared
 git clone https://github.com/uw-midsun/CAN-Explorer.git
 cd CAN-Explorer
 ```
 ## Requirements:
-The Django app requires Python >= 3.6 and MongoDB >= 3.6.3
+The Django app requires Python >= 3.6, Pipenv, MongoDB >= 3.6.3 and Redis >= 6.0
 Python3 should be installed in the Vagrant box already, but if you don't have MongoDB...
 
-## MongoDB setup
-To setup mongoDB for data collection run (only necessary the first time you start working with the database)
+## Dependencies setup (Mongo, Pipenv, Redis)
+To setup all dependencies for data collection run (only necessary the first time you start working with the database)
 ```
 bash setup.sh
```

Comment:
Let's make sure this setup.sh script is good before we remove everything else below from the readme